Deno is not available on all architectures Gleam is enabled for

--- a/test-output/src/tests/echo.rs
+++ b/test-output/src/tests/echo.rs
@@ -86,7 +86,7 @@
         insta::allow_duplicates! {
             assert_echo!(&snapshot_name, Some(Target::Erlang), None, $project_name);
             // assert_echo!(&snapshot_name, Some(Target::JavaScript), Some(Runtime::Bun), $project_name);
-            assert_echo!(&snapshot_name, Some(Target::JavaScript), Some(Runtime::Deno), $project_name);
+            // assert_echo!(&snapshot_name, Some(Target::JavaScript), Some(Runtime::Deno), $project_name);
             assert_echo!(&snapshot_name, Some(Target::JavaScript), Some(Runtime::NodeJs), $project_name);
         }
     };
@@ -96,7 +96,7 @@
         match $target {
             Target::JavaScript => insta::allow_duplicates! {
                 // assert_echo!(&snapshot_name, Some($target), Some(Runtime::Bun), $project_name);
-                assert_echo!(&snapshot_name, Some($target), Some(Runtime::Deno), $project_name);
+                // assert_echo!(&snapshot_name, Some($target), Some(Runtime::Deno), $project_name);
                 assert_echo!(&snapshot_name, Some($target), Some(Runtime::NodeJs), $project_name);
             },
             Target::Erlang => {
